files/build_templates/docker_image_ctl.j2

```diff
@@ -529,9 +539,58 @@ start() {
 {%- endif %}
 {%- if sonic_asic_platform == "mellanox" %}
     # TODO: Mellanox will remove the --tmpfs exception after SDK socket path changed in new SDK version
+{%- endif %}
+    if [ -z "$DEV" ]; then
+        pmon_sdk_sock="SX_API_SOCKET_FILE=/var/run/sx_sdk/sx_api.sock"
+        syncd_flag="-v"
+        syncd_var="/dev/shm:/dev/shm:rw"
+        ipc_var=""
+    else
+        pmon_sdk_sock="SX_API_SOCKET_FILE=/var/run/sx_sdk/sx_api_0.sock"
+        syncd_flag="--device"
+        syncd_var="/dev/sxdevs/sxcdev$(( $DEV + 1 )):/dev/sxdevs/sxcdev"
+        ipc_var="private"
+    fi
+{%- if docker_container_name == "snmp" %}
+# get snmp listening address and port list from redis
```

Comment:
># get snmp [](http://example.com/codeflow?start=0&length=10)

This block is not well formatted.

Reply:
removed

Reply:
From Line 544 to Line 573, this block is still not well formatted:
1. `if` not indented
2. use 2 spaces each indentation level, prefer 4 instead

Reply:
Still applicable.